src/unity/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -182,6 +183,10 @@ def create(dataset, target, feature, max_iterations=10,
 
     train_data = _tc.SFrame({'deep features': train_deep_features, 'labels': encoded_target})
     train_data = train_data.stack('deep features', new_column_name='deep features')
+    train_data, missing_ids = train_data.dropna_split(columns=['deep features'])
+
+    if len(missing_ids) > 0:
+        _logging.warning("Dropping %d examples which are less than 975ms in length." % len(missing_ids))
```

Comment:
In general this code looks good, but I'm wondering why we have this limitation -- is the underlying model not capable of handling smaller samples, or did we pick a hardcoded minimum sample size that seems to give good results?

Reply:
The underlying model is only capable of handling 975ms chunks. VGGish (the second part of the model classification pipeline) only accepts a fixed sized input to get that sized input the preprocessing code must have 975ms of audio.

Reply:
I think it's fine for now to drop smaller samples and warn, but I wonder if it would make sense (as a future enhancement) to either stretch or pad smaller samples so they are >= 975ms. (I'm not sure whether this actually works or not in terms of getting good results from the model, though.)

Reply:
I totally agree. I created #1712 to track this issue.